package.json

```diff
@@ -19,14 +19,15 @@
     }
   ],
   "scripts": {
-    "clean": "rm -rf ./lib/ && rm -rf ./dist/",
+    "clean": "rimraf ./lib/ && rimraf ./dist/",
     "compile": "npm run clean && npm run compile:lib && npm run compile:dist",
     "compile:lib": "coffee --compile -o ./lib/ ./src/coffee/card.coffee && node-sass ./src/scss/card.scss -o lib/ && replace '../scss/card.scss' './card.css' lib/card.js",
-    "compile:dist": "NODE_ENV=production webpack",
+    "compile:dist": "npm run env NODE_ENV=production && webpack",
```

Comment:
Hi Jesse, thanks for the feedback.
Yes, this is for windows support.
Normally a windows command prompt requires **SET** env_var=<value>
This seems to be a cross-platform compromise.